core/platform/datastore/cloud_datastore_services.py

```diff
@@ -222,15 +219,16 @@ def fetch_multiple_entities_by_ids_and_models(
         to the ids and models. The models corresponding to the same tuple in the
         input are grouped together.
     """
-    entity_keys = [] # type: List[Key]
+    entity_keys: List[Key] = []
     for (model_name, entity_ids) in ids_and_models:
         # Add the keys to the list of keys whose entities we have to fetch.
         entity_keys = (
             entity_keys +
             [ndb.Key(model_name, entity_id) for entity_id in entity_ids])
 
-    all_models = ndb.get_multi(entity_keys) # type: List[Optional[TYPE_MODEL_SUBCLASS]]
-    all_models_grouped_by_model_type = [] # type: List[List[Optional[TYPE_MODEL_SUBCLASS]]]
+    all_models: List[Optional[TYPE_MODEL_SUBCLASS]] = ndb.get_multi(entity_keys)
+    all_models_grouped_by_model_type: (
+        List[List[Optional[TYPE_MODEL_SUBCLASS]]]) = []
```

Comment:
Broken down the type annotation into multiple lines instead of that - preferring that because it would be similar to the case when a type is so long that it doesn't fit the line limit even after parenthesis break.